specification/intelligentdataexperiences/resource-manager/Microsoft.AadCustomSecurityAttributesDiagnosticSettings/preview/2023-03-01-preview/intelligentdataexperiences.json

```diff
@@ -0,0 +1,486 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-03-01-preview",
+    "title": "intelligentdataexperiences",
+    "description": "Azure Active Directory Custom Security Logs Client.",
+    "x-ms-code-generation-settings": {
+      "name": "intelligentdataexperiencesClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "security": [
+    {
+      "azure_auth": ["user_impersonation"]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.AadCustomSecurityAttributesDiagnosticSettings/operations": {
```

Comment:
>"/providers/Microsoft.AadCustomSecurityAttributesDiagnosticSettings/operations": { [](http://example.com/codeflow?start=4&length=82)

Please add pagination to this List API. See LintDiff errors in PR for more details.